frontend/scss/organisms/_o-custom-tours-viewer.scss

```diff
@@ -0,0 +1,126 @@
+// ------ Custom tours viewer
+
+// ---- Custom tours: Landing Page
+
+.aic-ct-landingpage__body {
+  padding-top: 32px;
+  padding-bottom: 32px;
+  background-color: $color__black--5;
+
+  &::before {
+    content: "";
+    position: absolute;
+    z-index: -1;
+    left: 50%;
+    top: 0;
+    bottom: 0;
+    width: 100vw;
+    margin-left: -50vw;
+    background-color: inherit;
+    pointer-events: none;
+  }
+
+  .title {
+    margin-bottom: 20px;
+  }
+
+  .aic-ct-grid-intro {
```

Comment:
Let's remove this from the nesting here if it doesn't need it

Reply:
Thank you, I've removed.